menu/cbs/menu_cbs_ok.c

```diff
@@ -1285,6 +1294,45 @@ static int generic_action_ok(const char *path,
             }
          }
          break;
+      case ACTION_OK_SIDELOAD_CORE:
+         {
+            settings_t            *settings = config_get_ptr();
+            char destination_path[PATH_MAX_LENGTH];
+            FILE *src, *dst;
+            char buffer[100];
+            int numr, numw;
+
+            fill_pathname_join(destination_path, settings->paths.directory_libretro, path_basename(action_path), sizeof(destination_path));
+            src = fopen(action_path, "rb");
+
+            if (!src)
```

Comment:
Would it be possible to use the VFS wrappers here instead?

Reply:
[vfs_implementation](https://github.com/libretro/libretro-common/blob/4b165e1d5edea374d5a56f3d477041560afb695e/include/vfs/vfs_implementation.h)?